explorer/src/pages/Attestations/index.tsx

```diff
@@ -62,7 +67,13 @@ export const Attestations: React.FC = () => {
   return (
     <TitleAndSwitcher>
       <DataTable columns={data.columns} data={data.list} link={APP_ROUTES.ATTESTATION_BY_ID} />
-      {attestationsCount ? <Pagination itemsCount={attestationsCount} handlePage={handlePage} /> : null}
+      {attestationsCount ? (
+        where ? (
+          <BasicPagination handlePage={handlePage} />
+        ) : (
+          <Pagination itemsCount={attestationsCount} handlePage={handlePage} />
+        )
+      ) : null}
```

Comment:
Instead of this nested ternary expression, can you extract at least
```
where ? (
          <BasicPagination handlePage={handlePage} />
        ) : (
          <Pagination itemsCount={attestationsCount} handlePage={handlePage} />
        )
```
to a function?

Reply:
fixed